Cesium.CodeGen/Ir/BlockItems/EnumeratorDefinition.cs

```diff
@@ -0,0 +1,6 @@
+using Cesium.CodeGen.Ir.Expressions;
+using Cesium.CodeGen.Ir.Types;
+
+namespace Cesium.CodeGen.Ir.BlockItems;
+
+internal record EnumeratorDefinition(string Identifier, IType Type, IExpression Value) : IBlockItem;
```

Comment:
_(Minor)_ The standard names them "enumeration constants", let's do the same.
```suggestion
internal record EnumConstantDefinition(string Identifier, IType Type, IExpression Value) : IBlockItem;
```